### PR TITLE
refine contributing doc

### PR DESCRIPTION
**Pull request by wonderflow**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,15 +28,22 @@ We also recommend you to learn about KubeVela's [design](docs/en/design.md) befo
 git clone git@github.com:oam-dev/kubevela.git
 ```
 
+KubeVela includes two parts, `vela cli` and `vela core`.
+
+- The `vela cli` is a command line tool that can build, run apps(with the help of `vela core`).
```
Comment (zzxwill) [inline CONTRIBUTING.md:33]: We can introduce `vela core` first as `vela cli` is referring to `vela core`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,15 +28,22 @@ We also recommend you to learn about KubeVela's [design](docs/en/design.md) befo
...
+- The `vela core` is actually a K8s controller, watch CRD and deploy resources.
```
Comment (zzxwill) [inline CONTRIBUTING.md:34]:
```suggestion
- The `vela core` is actually a K8s controller, which watches CRD and deploys resources.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,15 +28,22 @@ We also recommend you to learn about KubeVela's [design](docs/en/design.md) befo
...
+For local development, we probably need to build both of them.
+
 * Build Vela CLI
 
 ```shell script
 make
 ```
 
-* Configure vela to PATH
+After build successfully, `make` command will create `vela` binary to `bin/` under the project.
```
Comment (zzxwill) [inline CONTRIBUTING.md:44]:
```suggestion
After built successfully, `make` command will create `vela` binary to `bin/` under the project.
```
I think it means `After the Cli is built successfully`.

File: CONTRIBUTING.md (hunk 1 above)
Comment (wonderflow) [reply CONTRIBUTING.md:34]: yes thanks

**Review by ryanzhang-oss — APPROVED** [review]

**Pull request merged**